runtime/syntax/hgcommit.vim

```diff
@@ -1,21 +1,25 @@
 " Vim syntax file
-" Language:	hg (Mercurial) commit file
+" Language:	sl/hg (Sapling / Mercurial) commit file
 " Maintainer:	Ken Takata <kentkt at csc dot jp>
-" Last Change:	2012 Aug 23
-" Filenames:	hg-editor-*.txt
+"  		Max Coplan <mchcopl@gmail.com>
+" Last Change:	2022-12-08
+" Filenames:	hgcommit*.vim
```

Comment:
I meant that this syntax file will be applied to `hg-editor-*.txt`. Not sure if `Filenames:` is still needed when this supports sl.

Reply:
```suggestion
```